test/throughput/test_buffer_profile.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pytest
+
+from tianshou.data import (ListReplayBuffer, PrioritizedReplayBuffer,
+                           ReplayBuffer)
+
+
+@pytest.fixture(scope="module")
+def data():
+    np.random.seed(0)
+    obs = {'observable': np.random.rand(
+        100, 100), 'hidden': np.random.randint(1000, size=200)}
+    info = {'policy': "dqn", 'base': np.arange(10)}
+    add_data = {'obs': obs, 'rew': 1., 'act': np.random.rand(30),
+                'done': False, 'obs_next': obs, 'info': info}
+    buffer = ReplayBuffer(int(1e3), stack_num=100)
+    buffer2 = ReplayBuffer(int(1e4), stack_num=100)
+    Slice = slice(-3000, -1000, 2)
+    indexs = np.random.choice(int(1e3), size=3, replace=False)
+    return{
+        'add_data': add_data,
+        'buffer': buffer,
+        'buffer2': buffer2,
+        'Slice': Slice,
+        'indexs': indexs
+    }
+
+
+def test_init():
+    for _ in np.arange(1e5):
+        _ = ReplayBuffer(1e5)
+        _ = PrioritizedReplayBuffer(
+            size=int(1e5), alpha=0.5,
+            beta=0.5, repeat_sample=True)
+        _ = ListReplayBuffer()
```

Comment:
init of replaybuffer does very little. Why profile this?

Reply:
I tend to profile every core API of batch/buffer/collector because I cannot guarantee that in the future still costs very little time. However,  @Trinkle23897 has mentioned [similar question](https://github.com/thuml/tianshou/pull/143/files/d9981fb9c19f616613676fdf98ac3c80e01c6e1e#r460512028), so maybe I can remove those if you guys really consider it unnecessary. What do you think?

Reply:
As you like. More profiling is not a bad thing, anyway.